.github/workflows/sync-helpz.yml

```diff
@@ -0,0 +1,28 @@
+name: 'Sync mirrorz-help-ng'
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '33 22 * * *'  # 06:33 AM in Asia/Shanghai (UTC+8)
+jobs:
+  sync:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v4
+      with:
+        submodules: true
+    - name: Sync submodule
+      run: |
+        git submodule update --remote helpz/_posts
+        echo -n 'See content changes in: ' > /tmp/pr_msg
+        echo -n 'https://github.com/tuna/mirrorz-help-ng/compare/' >> /tmp/pr_msg
+        git diff --submodule=diff | head -n1 | cut -c24-39 >> /tmp/pr_msg
+    - name: Create Pull Request
+      uses: peter-evans/create-pull-request@v5
+      with:
+        title: 'helpz: sync mirrorz-help-ng'
+        commit-message: 'helpz: sync mirrorz-help-ng'
```

Comment:
And also include submodule hash here like `helpz: sync mirrorz-help-ng to xxxxxx`.

Or even better, we can somehow get the origin `mirrorz-docs` hash change from somewhere, e.g. a metadata file in `mirrorz-help-ng/transpiled`, or parsing the commit message